### PR TITLE
working on draft of get winner

### PR DESCRIPTION
**Pull request by Rachel311**
# Description

Fixes #

- What work was done?
- Why was this work done?
- What feature / user story is it for?
- Any relevant links or images / screenshots

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [x] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [x] No
- [ ] Not necessary

## Checklist

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by IsaacGHoward** [comment]
Instead of passing in a squadID of '1', wouldn't you want to take a number based on the squad ID that the child is in?

**Review by IsaacGHoward — CHANGES_REQUESTED** [review]
I would make it so that you could pass a squadId in instead of defaulting to 1

**Review by ashtilawat — CHANGES_REQUESTED** [review]
Let's take package-lock out of this PR. There seems to be 26K lines of code added to this PR. We want to keep the PR atomic.

File: src/components/common/GetWinnerButton.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+// import PropTypes from 'prop-types'
+
+const GetWinnerButton = props => {
+  const handleSubmit = e => {
+    e.preventDefault();
+    axios
+      .get('http://localhost:5000/api/game/results?squadId=1')
```
Comment (rikaciminieri) [inline src/components/common/GetWinnerButton.js:9]: I'm assuming the squad ID is hardcoded for now, just making note of it. Also we should probably be using the apiAuthGet() function to do the axios call to ensure those making the request are authenticated.

**Comment by rikaciminieri** [comment]
Good work overall! It would be helpful to fill out the description so that we can get an overview on what you worked on before looking at the code. Also, the title of the PR makes it sound like it isn't complete yet, but it seems like you completed the feature in the last commit.

**Pull request closed without merging**